# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/base/crashtests/1931933.html
# Commit: 9fa579e2d7b9
# Full Hash: 9fa579e2d7b97c7e6eaf77461d40fe94c06ba8d6
# Author: David Shin <dshin@mozilla.com>
# Date: 2024-11-21 04:47:28
# Description:
#   Bug 1931933: Use CSSMinMax instead of std::clamp in nsTextControlFrame::GetNaturalBaselineBOffset. r=layout-reviewers,emilio
#   
#   There is no guarantee that the block size is positive, since the size can overflow.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D229678
# ==============================================================================

diff -r 5ccabf34d30e -r 9fa579e2d7b9 layout/base/crashtests/1931933.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/base/crashtests/1931933.html	Wed Nov 20 19:49:42 2024 +0000
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<style>
+div {
+  display: grid;
+  padding: 33554400px;
+  border: 5px solid;
+}
+textarea {
+  padding: 33554400px;
+  border: 5px solid;
+}
+</style>
+<div class="container"><textarea></textarea></div>